src/sync/deployment.ts

```diff
@@ -49,12 +72,39 @@ const getTransformedDeployments = async (deployments, gitHubInstallationClient:
 	});
 
 	const transformedDeployments = await Promise.all(transformTasks);
-	return transformedDeployments
-		.filter(deployment => !!deployment)
+	return (transformedDeployments
+		.filter(deployment => !!deployment) as JiraDeploymentBulkSubmitData[])
 		.map(deployment => deployment.deployments)
 		.flat();
 };
 
+const saveDeploymentsForLaterUse = async (deployments: FetchDeploymentResponse["deployments"], gitHubBaseUrl: string, logger: Logger) => {
+	try {
+		const successDeployments: FetchDeploymentResponse["deployments"] = deployments.filter(d => (d.statuses?.nodes.some(n => n.state === "SUCCESS")));
```

Comment:
nit: same thing here with spacing. There's a lot of lines of code here and it gets really hard to separate the logic when it's all crammed together.

Reply:
fixed (to my best effort 🤷 )

Reply:
Love your work sir.